commands.js

```diff
@@ -410,6 +410,22 @@ let captureDom = (win) => {
         cy.writeFile(`./${vtConfFile.debug}/${imageName}-${imageType}/${imageName}.json`, dom);
     }
 };
+let scrollToTop = (win) =>{
+    let tries = 0;
+    let scrollOffset = 1;
+    do {
+        // Translate to the top of the page and then capture the dom
+        win.eval(`document.body.style.transform="translateY(0)"`);
+        cy.wait(250);
+        scrollOffset = win.eval(`scrollOffset = document.body.style.transform`);
+        tries += 1;
+    }while (scrollOffset = 0 && tries < 40)
```

Comment:
In JS you compare things with "==" and "===" this is assigning scrollOffset 0 instead of checking its value